ui/src/app/constants/texts.constants.ts

```diff
@@ -30,8 +30,9 @@ export const texts = {
 
   RUN_WORKFLOW_CONFIRMATION_TITLE: 'Run workflow',
```

Comment:
Are the `RUN_WORKFLOW` texts still used?

Reply:
Removed + removed service method and test for it

Reply:
What exactly did you remove? I was referring to `RUN_WORKFLOW_CONFIRMATION_TITLE` and `RUN_WORKFLOW_CONFIRMATION_CONTENT` :)

Reply:
RUN_WORKFLOW from api constants.
Fixed